### ISSUE TITLE
Embedded interactive viewers for high-level OpenAlex data

### ISSUE THREAD

**Issue by prototyperspective** [WDscholia/scholia#1812]
**Is your feature request related to a problem? Please describe.**
Scholia currently doesn't have much features to (visually and interactively) explore the big picture / bird's eye view of scientific research, such as developments in:

* Science itself in general (e.g. "increasing interdisciplinarity")
* Data about the types and forms of scientific papers (such as the share of papers published as open access)
* Generally metascience-relevant aggregated metadata about science-related wikidata contents
* Developments within a scientific field (such as changes related to topics; e.g. growth of research or reviews about certain topics etc)
* Developments on the level of scientific fields

A more concrete example is below.

**Describe the solution you'd like**
OpenAlex has such data and makes it queriable. I asked if somebody could create an image or interactive/queriable visualizations of year-spanning changes to science overall such as the share of papers and authors by field on Twitter. This helped lead to [Solution 1](https://twitter.com/h_karlstrom/status/1490998751296892928) and [Solution 2](https://twitter.com/TheShubhanshu/status/1490916221625466883). There's also code for creating these visualizations.

Of course the data is limited by what's in the database (the papers and the metadata these papers currently have), but the database is probably growing over time (in terms of both papers and metadata), is probably already large enough and even if not it's useful and interesting regardless.

Maybe this could also be added as interactive graphs into Wikipedia so if there is an easy-to template for these interactive graphs please let me know.

Such interactive visualization queries could be built collaboratively and then made available on the relevant pages on Scholia. Pages about topics (or scientific field) could for example show the development of the number / shares of authors per top-level subtopic of the topic (or scientific field).

**Describe alternatives you've considered**

**Additional context**

**Comment by fnielsen** [comment]
Yes, it could be interesting to have overviews, both for scientometrics and for navigation. Queries I can imaging would unfortunately be quite expensive. So I suppose one would perhaps need to rely on another endpoint/service with more gracious timeout.
